packages/shared/src/components/cards/ad/AdList.tsx

```diff
@@ -42,21 +44,19 @@ export const AdList = forwardRef(function AdCard(
     <FeedItemContainer
       domProps={domProps}
       ref={ref}
-      flagProps={{
-        adAttribution: (
-          <AdAttribution ad={ad} className={{ typo: 'typo-caption1' }} />
-        ),
-        type: undefined,
-      }}
       data-testid="adItem"
       linkProps={getLinkProps({ ad, onLinkClick })}
     >
       <CardContent>
-        <CardTitle className="mr-4 line-clamp-4 flex-1 font-bold typo-title3">
+        <CardTitle className="!mt-0 mr-4 line-clamp-4 flex-1 font-bold typo-title3">
           {ad.description}
         </CardTitle>
         <AdImage ad={ad} ImageComponent={CardImage} className="mt-4" />
       </CardContent>
+      <div className="z-1 flex items-center pt-4">
+        <AdAttribution ad={ad} />
+        <RemoveAd />
```

Comment:
@omBratteng This is not behind the feature flag now

Reply:
🙈

Reply:
https://github.com/dailydotdev/apps/pull/3842